Gems/PhysX/Code/Source/Pipeline/HeightFieldAssetHandler.h

```diff
@@ -18,7 +18,7 @@ namespace PhysX
     {
         struct HeightFieldAssetHeader
         {
-            AZ::u32 m_assetVersion = 2;
+            AZ::u32 m_assetVersion = 1;
```

Comment:
Going back a version? Would that conflict with users having old assets with version 1 already?

Reply:
This whole code has been unused since Lumberyard (surprisingly not deleted!), so there are no assets exist, hence starting from the beginning - version 1